### ISSUE TITLE
Provider not compatible with Terraform 0.12

### ISSUE THREAD

**Issue by stepanstipl** [DeviaVir/terraform-provider-gsuite#50]
The current version of the provider is not compatible with Terraform 0.12 as the plugin API changed, see https://www.terraform.io/upgrade-guides/0-12.html#upgrading-terraform-providers. It might be just a matter of rebuilding with new version of `github.com/hashicorp/terraform`.

**Comment by DeviaVir** [comment]
Can you try with `v0.1.15`? I just rebuilt it.

**Comment by stepanstipl** [comment]
Thanks, but still no luck, the message is:

```console
Error: failed to load provider "gsuite": Incompatible API version with plugin. Plugin version: 4, Client versions: [5]
```

I can see that the terraform version in `Gopkg.lock` is still `v0.11.3`. Latest stable v0.11 is `v0.11.13` atm. I've tried rebuilding with that one, which went fine, but did not solve the problem.

I believe it's necessary to update to v0.12, atm. `0.12.0-beta1`, which should result in provider being compatible both with old (0.11) and new (0.12) Terraform API, but unfortunately this results in some build errors (I've tried both updating just `github.com/hashicorp/terraform`, as well as updating everything with `dep ensure -update`, but both results in some build errors).
